src/dataSources.ts

```diff
@@ -110,7 +111,7 @@ export class FileUploadDataSource extends RemoteGraphQLDataSource {
 
       return response;
     } catch (error) {
-      this.didEncounterError(error, httpRequest);
+//      this.didEncounterError(error, fetchRequest, httpResponse, context);
```

Comment:
Is this commented out intentionally? If so, it would be nice if it had some comment on why that has been done 🙂

Reply:
My mistake, 'didEncounterError' has changed and I didn't do the change properly.
Anyway [didEncounterError](https://github.com/apollographql/federation/blob/f958fb964f9baa53edd149bcdf0d1217e7c39f0c/gateway-js/src/datasources/RemoteGraphQLDataSource.ts#L272) just throw error